.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-select = E901,E999,F821,F822,F823
```

Comment:
@karlhigley I removed those from here, so there might be cases when a developer changes a ```.py``` file it would see som warning according to flak8. Is this alright?